src/styles/blocks/top-row-layout.scss

```diff
@@ -0,0 +1,33 @@
+.top-row-layout {
+
+  &--one-icon {
+    @include headerNavigationGrid(1, 18px, 30px);
+    @include onTablet {
+      @include headerNavigationGrid(1, 24px, 63px);
+    }
+  }
+
+  &--two-icons {
+    @include headerNavigationGrid(2, 18px, 30px);
```

Comment:
Consider using a more descriptive mixin name instead of 'headerNavigationGrid'. For example, 'topRowLayoutGrid'.